### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by pradipgharge**

### CODE REVIEW

**Review by mehulsatardekar — APPROVED** [review]
didn't find any issues. from my side its ready to merge

File: css/style.css [hunk 1]
```diff
@@ -9,6 +9,7 @@ body{
     background-color: var(--solid-black-color);
     padding: 0.8rem 0;
 }
+
 .navbar{
     display: flex;
```
Comment (parvez-rashid) [inline css/style.css:14]: Have noticed `flex` properties being repeated a lot, you can use define a utility class for this and use it accordingly.

**Review by parvez-rashid — APPROVED** [review]
UI review: I like the UI, it's clean and minimalistic. I have a comment below. Other than that, code overall is well-structured, can be merged.

**Review by sri-rishi — APPROVED** [review]
Great work, couldn't find any issue feel free to merge

**Review by rahulk31 — APPROVED** [review]
The code looks well structured to me. May be you can plan to have some utility classes for flex properties in future.

Really liked the UI, awesome work there. You are good to merge this 👍

File: css/style.css (hunk 1 above)
Comment (pradipgharge) [reply css/style.css:14]: Thanks for the review, will add them after completing P0 (keeping in view all the files).

**Review by eakansh125 — APPROVED** [review]
I could not find any major syntactic issue looking at the code. Looks neat, even rems are at place. Good to be merged from my side.

**Pull request merged**